contracts/SpokePool.sol

```diff
@@ -662,20 +669,17 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
 
     // This function is isolated and made virtual to allow different L2's to implement chain specific recovery of
     // signers from signatures because some L2s might not support ecrecover, such as those with account abstraction
-    // like ZKSync.
+    // like ZKSync. To be safe, consider always reverting this function for L2s where ecrecover is different from how
```

Comment:
Does ZKSync not support standard ecrecover?

Reply:
I guess they do now, but its a special case so we need to be careful when integrating with zksync: https://v2-docs.zksync.io/dev/testnet/status.html#currently-supported-features